src/MPAGSCipher/CaesarCipher.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef MPAGSCIPHER_CAESARCIPHER_HPP
+#define MPAGSCIPHER_CAESARCIPHER_HPP
+
+#include <string>
+#include <vector>
+#include <iostream>
+#include "CipherMode.hpp"
+
+class CaesarCipher {
+  public:
+    //Members
+    std::string applyCipher(const std::string in_str, const CipherMode mode);
+    
+
+    //Constructors
+    explicit CaesarCipher(const std::size_t key);
+    explicit CaesarCipher(const std::string key_string);
```

Comment:
Better to take `std::string` objects by reference:
```suggestion
    std::string applyCipher(const std::string& in_str, const CipherMode mode);
    

    //Constructors
    explicit CaesarCipher(const std::size_t key);
    explicit CaesarCipher(const std::string& key_string);
```